docs/content/en/integration/openid-connect/budibase/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Budibase"
+description: "Integrating Budibase with the Authelia OpenID Connect 1.0 Provider."
+lead: ""
+date: 2023-11-12T15:50:35+00:00
+draft: false
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 620
+toc: true
+community: true
+---
+
+## Tested Versions
+
+- [Authelia]
+  - [v4.37.5](https://github.com/authelia/authelia/releases/tag/v4.37.5)
+- [Budibase]
+  - 2.13.9
+
+## Before You Begin
+
+{{% oidc-common %}}
+
+### Authelia
+
+Authelia configuration.yml
+
+```yaml
+identity_providers:
+  oidc:
+    clients:
+      - id: budibase
+        secret: mysecret
+        authorization_policy: two_factor
+        redirect_uris:
+          - https://budibase.example.com/api/global/auth/oidc/callback
+        scopes:
+          - openid
+          - profile
+          - email
+          - offline_access
+```
+
+## Budibase
+
+### Organization configuration
+
+Go on the builder main page: **Settings > Organization** or url : https://budibase.justmammoth.us/builder/portal/settings/organisation
+
+{{< figure src="budibase_org.png" alt="Budibase" width="300" >}}
```

Comment:
Instead of masking the images change the values in the UI to the values in the example and take them again. This will help avoid user confusion in this area.